pkg/api/v1alpha1/vspheremachineconfig.go

```diff
@@ -140,3 +140,11 @@ func validateVSphereMachineConfig(config *VSphereMachineConfig) error {
 
 	return nil
 }
+
+func validateVSphereMachineConfigHasTemplate(config *VSphereMachineConfig) error {
```

Comment:
Is there a more appropriate place to put this?

Reply:
I think it makes sense here since it's where we have the other validations